.travis.yml

```diff
@@ -56,14 +56,13 @@ script:
   - export TEST_EXTRA=""
   - if [[ "$COVERAGE" == "true" ]]; then
       export TEST_EXTRA="--cov-config .coveragerc --cov nengo";
-    fi
-  - if [[ "$STATIC" == "false" ]]; then
+    elif [[ "$STATIC" == "true" ]]; then
```

Comment:
Yeah, my heuristic is what will be executed most often ;) I think it's partly because the 'if' is assumed to be true for branch prediction way way under the hood, which is probably not relevant here, but I definitely read the if before the else, so it works in that way too.